app/static/openapi.yaml

```diff
@@ -398,6 +398,7 @@ paths:
                   times_clicked: 0
                   upvotes: 0
                   url: 'http://www.freetechbooks.com/'
+                  user_vote_direction: null
```

Comment:
Thanks for adding the `user_vote_direction` in the examples. We'll also need to add it (and a description) to the ResourceData schema
https://github.com/OperationCode/resources_api/blob/b2149912b71c6fe04daa4901d75c2a825899c0f6/app/static/openapi.yaml#L1061-L1100